cli/tools/run.rs

```diff
@@ -102,8 +103,12 @@ async fn run_with_watch(flags: Flags, script: String) -> Result<i32, AnyError> {
   let flags = Arc::new(flags);
   let main_module = resolve_url_or_path(&script)?;
   let (sender, receiver) = tokio::sync::mpsc::unbounded_channel();
-  let mut ps =
-    ProcState::build_for_file_watcher((*flags).clone(), sender.clone()).await?;
+  let mut ps = ProcState::build_for_file_watcher(
+    (*flags).clone(),
+    main_module.clone(),
```

Comment:
Could be derived from `flags` so I don't think we need this.

Reply:
Changed by adding `CliOptions::resolve_main_module()`